lib/AST/DiagnosticEngine.cpp

```diff
@@ -302,6 +302,70 @@ InFlightDiagnostic::fixItReplaceChars(SourceLoc Start, SourceLoc End,
   return *this;
 }
 
+SourceLoc
+DiagnosticEngine::getBestAddImportFixItLoc(const Decl* Member) const {
```

Comment:
Nit: Formatting here is a bit off. Maybe `getBestLocationForMissingImport`?

Reply:
Thanks, reformatting :)